### ISSUE TITLE
Mumble make random "Hugh" or "Beep" sound as long as the speaker is on

### ISSUE THREAD

**Issue by tianxuanliu** [mumble-voip/mumble#5921]
### Description

Hi everyone,

It's hard to describe. Mumble make random "beep" or "hugh" sound when the speaker is turned on, every 5 or 10 sec. The sound is like the old NES game, someone fires a laser gun. 

Recording: https://youtu.be/c5X4FdMsddg

It happens on computer speakers and headphones.
It happens no matter if I select to push to talk or other methods. 
It happens if I have the microphone turned on.

As long as the speaker is on, it makes that noise. 
Let me know if you need more info or screen recording. 

Thanks.

### Steps to reproduce

Nothing special. 

### Mumble version

All 1.4 versions

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

**Comment by davidebeatrici** [comment]
A recording of the specific sound would definitely be helpful.

**Comment by tianxuanliu** [comment]
Update recording: https://youtu.be/c5X4FdMsddg
The sound is annoying and also hilarious. lol

**Comment by ppfeufer** [comment]
That's the warning sound when your microphone is muted but detects sound input to transmit to Mumble ...

I think you can deactivate that somewhere in the settings

**Comment by ppfeufer** [comment]
Yep, you can deactivate it, it's this one (see image)

![image](https://user-images.githubusercontent.com/2989985/194730996-fcb1589c-8133-469a-8b1f-3690349324c1.png)

**Comment by tianxuanliu** [comment]
Wow! Thanks, Peter. 
To all: I think it is better to set this warning off by default. Or change the warning sound to something more easily understandable by users.

**Comment by Krzmbrzl** [comment]
If it's off by default, it will be close to useless as I don't think anyone checks all settings to see what might be available. It being on by default makes sure everyone is aware of the existence of this feature and can then opt out if they don't like it.

As for the sound: I'm sure it could use an improvement but so far nobody has come up with a better sound :shrug:

**Comment by carlosm2** [comment]
thanks for this solution, it never happened to me before, I've been a Debian user for many years and now I got this "beep" on an apple silicon M2 with MacOS Ventura 13.3.1 but with this solution it worked perfectly, thanks https://github.com/mumble-voip/mumble/issues/5921#issuecomment-1272412221

**Comment by w3us** [comment]
Many thanks for pointing us to the "cue" dialogue.  Man! that is one annoying sound.

**Comment by L3viathan** [comment]
> It being on by default makes sure everyone is aware of the existence of this feature

Not really, it just means everyone is affected by it. It's not very discoverable what this feature _is_, I had to google and find this issue.

Perhaps some kind of notification the first time it happens would be good?

**Comment by Krzmbrzl** [comment]
> Perhaps some kind of notification the first time it happens would be good?

This has already been implemented via #6155 :)

**Comment by chris451gh** [comment]
I use mumble 24x7 for shipping audio around the world. It gets through more networks without trouble. I have 12 PC's
in the chat for a group sound connection and mumble allows a mixer effect, so any source comes through depending on what channel or mute they are on.. 
One of the computers had this beep boop noise almost constantly. 
Eventually I discovered which of the multiple computers it was coming from. 
Each one is set up with virtual audio cable, and default audio device is vb cd 
On that PC I went through the message sounds  and disabled everything.
To prove where it was coming from killed all apps one at a time, til it disappeared with mumble.
I found the solution here in the mumble-voip issues. It was the cue thing. Even though thresholds were well above the transmit
vox threshold, this feature was reacting to low level noise  
Here I saw the image <cue checkbox>
It probably got clicked to figure out what it does. and was not noticed when it was done as the source audio is fed to other
mumble clients, so it did not seem to do anything. 
I'd say get rid of this optional noise, who uses it for anything? 
Do a revision of the user instructions to include what cue does and what its for.
It transmits this noise to other clients, who do not know what its for but may see who its from.

In volume control it says Name Not available on mixer for this source. If I mute that or mute mumble on the mixer it goes away.

It took a lot of digging to find out where this noise was coming. I don't see any use  for it. It duplicates capabilities elsewhere
